server/src/test/java/com/broadcom/lsp/cobol/positive/CobolTextHelper.java

```diff
@@ -0,0 +1,31 @@
+package com.broadcom.lsp.cobol.positive;
+
+import lombok.experimental.UtilityClass;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+@Slf4j
+@UtilityClass
+public class CobolTextHelper {
+    public CobolText fromStream(String name, InputStream stream) {
+        String fullText = null;
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(stream));
+            StringBuilder builder = new StringBuilder();
+
+            for (String line = bufferedReader.readLine();
+                 line != null;
+                 line = bufferedReader.readLine()) {
+                builder.append(line).append("\r\n");
+            }
+            fullText = builder.toString();
+        } catch (IOException e) {
+            LOG.error(e.getMessage());
```

Comment:
As I see, for error case, it is still returning the object, and once used below no validation there. Better return Optional object and check ifPresent..

Reply:
I have just moved this code from one class to another, but I agree it can be re-implemented better 👍